templates/_formhelpers.html

```diff
@@ -1,11 +1,11 @@
-{% macro render_field(field, label_visible=true) -%}
-     <div class="form-group {% if field.errors %}has-error{% endif %} {{ kwargs.pop('class_', '') }}">
+{% macro render_field(field, label_visible=true, required=false) -%}
```

Comment:
I don't think you need to pass a flag, each field has a flag for required already. `field.flags.required`

Reply:
Thank you, great find! https://github.com/bloomberg/pybossa-default-theme/pull/214/commits/8954cc49372345b47b7066176624317048914086